source/dotnet/DomainTests/DomainTests.csproj

```
@@ -7,22 +7,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Energinet.DataHub.Core.FunctionApp.TestCommon" Version="3.5.0" />
-    <PackageReference Include="Energinet.DataHub.Wholesale.Client" Version="3.4.0" />
+    <PackageReference Include="Energinet.DataHub.Core.FunctionApp.TestCommon" Version="4.0.1" />
+    <PackageReference Include="Energinet.DataHub.Wholesale.Client" Version="6.0.1" />
```

Whoops. Not 6.0.1. My bad.
```suggestion
    <PackageReference Include="Energinet.DataHub.Wholesale.Client" Version="6.0.0" />
```